src/models/auditModelGenerator.js

```diff
@@ -0,0 +1,149 @@
+// Load in our dependencies
+import { Model, DataTypes } from 'sequelize'; // eslint-disable-line import/no-import-module-exports
+import httpContext from 'express-http-context'; // eslint-disable-line import/no-import-module-exports
+import { auditLogger } from '../logger'; // eslint-disable-line import/no-import-module-exports
+
+const dmlType = ['INSERT', 'UPDATE', 'DELETE'];
+
+// const exception = () => {
+//   throw new Error(
+//     'Audit log only allows reading and inserting data,'
+//     + ' all modification and removal is not allowed.'
+//   );
+// };
+
+const tryJsonParse = (data) => {
+  let newData = data;
+  if (data) {
+    if (typeof data === 'object') {
+      Object.entries(data).forEach(([key, value]) => {
+        newData[key] = tryJsonParse(value);
+      });
+    } else if (typeof data === 'string') {
```

Comment:
Do we need to consider if the data is a number or bool?

Reply:
numbers and bools work as expected